plugins/content/pagenavigation/pagenavigation.php

```diff
@@ -127,8 +127,8 @@ public function onContentBeforeDisplay($context, &$row, &$params, $page = 0)
 			}
 
 			$xwhere = ' AND (ws.condition = 1 OR ws.condition = -2)'
-				. ' AND (publish_up = ' . $db->quote($nullDate) . ' OR publish_up <= ' . $db->quote($now) . ')'
-				. ' AND (publish_down = ' . $db->quote($nullDate) . ' OR publish_down >= ' . $db->quote($now) . ')';
+				. ' AND (publish_up = IS NULL OR publish_up <= ' . $db->quote($now) . ')'
```

Comment:
Need to remove the `=` here should be `publish_up = IS NULL` and publish down on line below